### ISSUE TITLE
small question on cifar example

### ISSUE THREAD

**Issue by bluedlw** [vlfeat/matconvnet#707]
In the file cnn_cifar.m, 
should it be W = z(:,set == 1)*z(:,set == 1)'/50000  instead of  W = z(:,set == 1)*z(:,set == 1)'/60000?
Since training set is 50000, and validation set is 10000.

**Comment by fengyunxiaozi** [comment]
the test dataset should be pre-processed the same as the train dataset
